File: Iniciante/1154 - Idades/Python/1154.py
def main():
    number_of_ages = 0
    sum_of_ages = 0

    while True:
        age = int(input())

        if age < 0:
            break

        number_of_ages += 1
        sum_of_ages += age

    average_ages = sum_of_ages / number_of_ages
    print(f"{average_ages:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1154 - Idades/JavaScript/1154.js
function main() {
    const fs = require('fs');
    const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

    let numberOfAges = 0;
    let sumOfAges = 0;

    for (let i = 0; i < input.length; i++) {
        let age = parseInt(input[i]);

        if (age < 0) {
            break;
        }

        numberOfAges++;
        sumOfAges += age;
    }

    let averageAges = sumOfAges / numberOfAges;
    console.log(averageAges.toFixed(2));
}

main();